lib/src/controls/navigation/navigation_view/pane.dart

```diff
@@ -61,6 +61,11 @@ enum PaneDisplayMode {
   auto,
 }
 
+enum PanePosition {
+  left,
+  right,
+}
```

Comment:
This is nice, but doesn't work well in a directionality context. The available options should be `start` and `end`, so that it'd work well with rtl languages

Also, documentation for these is missing